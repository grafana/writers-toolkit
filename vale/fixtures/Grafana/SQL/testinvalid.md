Don't use:

- A SQL database
- a SQL database
- a SQL server

Also don't use:

- An SQL Server
- an SQL Server
