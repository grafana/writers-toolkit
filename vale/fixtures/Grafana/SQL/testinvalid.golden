testinvalid.md:3:3:Grafana.SQL:Use 'an SQL' instead of 'A SQL'. The article—a or an—that you use before the acronym SQL depends on how the word is pronounced. When referring to the product Microsoft SQL Server, SQL should be pronounced "sequel". In this case, use the article 'a', as in "a SQL Server analysis". When referring to the term in any other context, such as SQL databases, errors, or servers, SQL should be pronounced "ess-cue-el". In this case, use the article 'an', as in "an SQL error". 
testinvalid.md:4:3:Grafana.SQL:Use 'an SQL' instead of 'a SQL'. The article—a or an—that you use before the acronym SQL depends on how the word is pronounced. When referring to the product Microsoft SQL Server, SQL should be pronounced "sequel". In this case, use the article 'a', as in "a SQL Server analysis". When referring to the term in any other context, such as SQL databases, errors, or servers, SQL should be pronounced "ess-cue-el". In this case, use the article 'an', as in "an SQL error". 
testinvalid.md:5:3:Grafana.SQL:Use 'an SQL server' or 'a SQL Server' instead of 'a SQL server'. The article—a or an—that you use before the acronym SQL depends on how the word is pronounced. When referring to the product Microsoft SQL Server, SQL should be pronounced "sequel". In this case, use the article 'a', as in "a SQL Server analysis". When referring to the term in any other context, such as SQL databases, errors, or servers, SQL should be pronounced "ess-cue-el". In this case, use the article 'an', as in "an SQL error". 
testinvalid.md:9:3:Grafana.SQL:Use 'a SQL Server' instead of 'An SQL Server'. The article—a or an—that you use before the acronym SQL depends on how the word is pronounced. When referring to the product Microsoft SQL Server, SQL should be pronounced "sequel". In this case, use the article 'a', as in "a SQL Server analysis". When referring to the term in any other context, such as SQL databases, errors, or servers, SQL should be pronounced "ess-cue-el". In this case, use the article 'an', as in "an SQL error". 
testinvalid.md:10:3:Grafana.SQL:Use 'a SQL Server' instead of 'an SQL Server'. The article—a or an—that you use before the acronym SQL depends on how the word is pronounced. When referring to the product Microsoft SQL Server, SQL should be pronounced "sequel". In this case, use the article 'a', as in "a SQL Server analysis". When referring to the term in any other context, such as SQL databases, errors, or servers, SQL should be pronounced "ess-cue-el". In this case, use the article 'an', as in "an SQL error". 
